1-JavaScript/Week3/homworkSolition/ex1-doubleEvenNumbers.test.js

```diff
@@ -0,0 +1,13 @@
+
+function doubleEvenNumbers(numbers) {
+
+  const doubledEvennumber = numbers.filter((even)=>{ return even % 2 == 0 })
+  .map((num)=>{ return num * 2 })
+  return doubledEvennumber;
+}
```

Comment:
Nice work! 💪 Some tips for improvement:

1. You can "implicitly return" from the functions (see my suggestion below)
2. Everything can be returned in one line so you don't need to create separate variables. This is however a **preference**, you don't have to change this because you can definitely argue that your code is more readable 😄 

```suggestion
function doubleEvenNumbers(numbers) {
  return numbers.filter((even) => even % 2 == 0).map((num) => num * 2)
}
```